tests/unit/objects/test_projects.py

```diff
@@ -655,11 +655,6 @@ def test_delete_project(project, resp_delete_project):
     project.delete()
 
 
-@pytest.mark.skip(reason="missing test")
-def test_restore_project(gl):
-    pass
```

Comment:
The `restore()` function is currently not implement for `projects` as far as I can see 🤔

Reply:
Oh, oops, not sure how it got into the test stubs then :) Thanks, opened https://github.com/python-gitlab/python-gitlab/issues/2070 for this!

Reply:
yeah we don't really prioritize premium features.